media-sound/qmmp/qmmp-2.2.3.ebuild

```diff
@@ -0,0 +1,201 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# docs should be migrated to cmake
+DOCS_BUILDER=doxygen
+DOCS_DIR="${S}"/doc
+
+inherit cmake docs xdg
```

Comment:
why use eclass docs if doc project exists upstream ?

Reply:
See comments below. But this is an API/dev doc no ?
It needs also a doc useflag and bdepend with doxygen if you revert to cmake